README.md

```diff
@@ -68,10 +69,18 @@ If you want to set up a cronjob, send the output to a temp file and when finishe
 file over the "actual" file. Otherwise you might end up with an incomplete page when the checks are 
 running. Like so:
 
-	* * * * * /bin/bash /opt/srvmon/srvmon.sh > /var/www/index.html.tmp && /bin/mv /var/www/index.html.tmp /var/www/index.html
+	* * * * * /bin/bash /opt/srvmon/srvmon > /var/www/index.html.tmp && /bin/mv /var/www/index.html.tmp /var/www/index.html
 
 If the check fails for whatever reason, the "old" page will not be overridden.
 
+### CGI mode
+
+If you want to set up CGI mode, you need to copy the script to your server CGI directory. 
```

Comment:
Same here as the comment on the script cgi variable. I saw the reddit thread where this was discussed but a random user might not know what cgi is or why they would want that. Could you please add a few lines explaining what it is and the benefits or use cases?

Reply:
Added

Reply:
Thanks!